nextmv/nextmv/cloud/application.py

```diff
@@ -1280,6 +1372,53 @@ def new_batch_experiment_with_result(
 
         return self.batch_experiment_with_polling(batch_id=batch_id, polling_options=polling_options)
 
+    def new_ensemble_defintion(
+        self,
+        id: str,
+        run_groups: list[RunGroup],
+        rules: list[EvaluationRule],
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+    ) -> EnsembleDefinition:
+        """
+        Create a new ensemble definition.
+
+        Parameters
+        ----------
+        id: str
+            ID of the ensemble defintion.
+        name: Optional[str]
+            Name of the ensemble definition.
+        description: Optional[str]
+            Description of the ensemble definition.
+        run_groups: Optional[list[RunGroup]]
+            Information to facilitate the execution of child runs.
+        rules: Optional[list[EvaluationRule]]
+            Information to facilitate the selection of
+            a result for the ensemble run from child runs.
+        """
+
+        if name is None:
+            name = id
+        if description is None:
+            description = name
+
+        payload = {
+            "id": id,
+            "run_groups": [run_group.to_dict() for run_group in run_groups],
+            "rules": [rule.to_dict() for rule in rules],
+            "name": name,
+            "description": description,
+        }
+
+        response = self.client.request(
+            method="POST",
+            endpoint=f"{self.endpoint}/ensembles",
```

Comment:
Nit: consider defining `ensembles` as an attribute of the class, similar to `experiments_endpoint`:

```python
experiments_endpoint: str = "{base}/experiments"
```

That way, you can use `{self.ensembles_endpoint}` as opposed to `{self.endpoint}/ensembles` here and in other places.